packages/ai/core/generate-object/stream-object.ts

```diff
@@ -100,44 +100,44 @@ functionality that can be fully encapsulated in the provider.
   providerMetadata: ProviderMetadata | undefined;
 }) => Promise<void> | void;
 
-/**
-Generate a structured, typed object for a given prompt and schema using a language model.
-
-This function streams the output. If you do not want to stream the output, use `generateObject` instead.
-
-@return
-A result object for accessing the partial object stream and additional information.
- */
-export function streamObject<OBJECT>(
+export function streamObject<
```

Comment:
How will this impact jsdoc hovers?

Reply:
(the function description seems to have been removed)

Reply:
done 👍